src/migratus/cli.clj

```diff
@@ -163,6 +163,11 @@
         (log/info "Missing config file" (.getMessage e)
                   "\nYou can use --config path_to_file to specify a path to config file")))))
 
+(defn requires-mig-id [cfg action [_ & args]]
```

Comment:
This does not look ok.
`requires-mig-id` does not make sense IMO. 

have `up` and `down` functions that each do their own validation.

Reply:
absolutely right :)
added both validation fn